controller/instance_handler.go

```diff
@@ -158,6 +162,9 @@ func (h *InstanceHandler) ReconcileInstanceState(obj interface{}, spec *types.In
 			logrus.Errorf("%v", err)
 			return err
 		}
+	} else if status.CurrentState == types.InstanceStateError && pod != nil {
+		logs, err := h.getPodLogs(pod.Name, CrashLogsTaillines)
+		logrus.Warnf("instance %v crashed, error: \n%v\nlogs: \n%v", pod.Name, err, logs)
```

Comment:
If err is set, it doesn't make sense to print log which is invalid.

If err is nil, it doesn't need to be printed out.

```
If err != nil {
   logrus.Warnf("instance %v crashed, log:xxx")
} else {
   logrus.Warnf("instance %v crashed, but cannot get log, error %v", xxx)
}
```